WalletWasabi/JsonConverters/KeyPathJsonConverter.cs

```diff
@@ -12,8 +12,12 @@ public override bool CanConvert(Type objectType)
 	}
 
 	/// <inheritdoc />
-	public override object? ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
+	public override object? ReadJson(JsonReader reader, Type objectType, object? existingValue, JsonSerializer serializer)
 	{
+		if (reader.Value is null)
+		{
+			throw new ArgumentNullException("Can't read json, value was null.");
```

Comment:
Ultra nit, Visual Studio cries over CA2208:

```suggestion
						throw new ArgumentNullException(nameof(reader.Value), "Can't read json, value was null.");

```

Reply:
#7587